### DISCUSSION TITLE
NFC for yubikey challenge response?

### DISCUSSION THREAD

**Discussion by ArtemisDc** [keepassxreboot/keepassxc#10844] [Q&A] [answered]
Hey,
I am considering getting a NFC Pad for my pc and a laptop with NFC for the yubikey.
Does keepassxc support challenge response via nfc? I haven't read anything about NFC support and am not really sure how much the yubikey via nfc differs from getting used with usb. 

Thanks for your answers :)

**Comment by droidmonkey** [comment, marked as answer]
Yes we do, it works the same as USB for us
